docs/en/cloud/security/gcp-private-service-connect.md

```diff
@@ -0,0 +1,224 @@
+---
+slug: /en/manage/security/gcp-private-service-connect
+sidebar_label: GCP Private Service Connect
+title: Setting up GCP Private Service Connect
+---
+import GCPRegions from '@site/docs/en/_snippets/_gcp_regions.md';
+
+## Private Service Connect
+
+Private Service Connect is a capability of Google Cloud networking that allows consumers
+to access managed services privately from inside their VPC network. Similarly, it allows
+managed service producers to host these services in their own separate VPC networks and
+offer a private connection to their consumers.
+
+Service producers publish their applications to consumers by creating Private Service
+Connect services. Service consumers access those Private Service Connect services directly
+through one of these Private Service Connect types.
+
+![Overview of PSC](@site/docs/en/cloud/security/images/gcp-psc-overview.png)
+
+:::important
+By default a ClickHouse service is not available via Private Service connect even if the
+PSC connection is approved and established; you need explicitly add the PSC ID to the 
+allow list on an instance level by creating a Support request.
+
+The support request will be covered later in this document.
+:::
+
+
+## Supported regions
+
+<GCPRegions/>
+
+Cross-region connectivity is not supported-producer and consumer regions should be the same.
+You will be able to connect from other regions within your VPC if you enable Global access
+on the PSC level (see below).
+
+## Setting up PSC
+### Adding a PSC Connection
+
+In the Google Cloud console, navigate to **Network services -> Private Service Connect**
+
+![Open PSC](@site/docs/en/cloud/security/images/gcp-psc-open.png)
+
+Open the Private Service Connect creation dialog by clicking on the **Connect Endpoint** button.
+
+
+- **Target**: Please use **Published service**
```

Comment:
@bianpengyuan should this be the entry from the `Service Attachment` column in the table instead of `Published service`?

Reply:
Yes, I made a new screen shot of this:
![Screenshot 2023-06-07 at 10 59 33 AM](https://github.com/ClickHouse/clickhouse-docs/assets/34738376/1e341064-284f-43ea-9fba-351ef745fec4)